docs/tutorials/offpolicy-tutorial.md

```diff
@@ -252,10 +294,19 @@ updates.
 If everything goes well, the training success should quickly reach values around 0.7-0.8 on GPU and converge to values
 close to 1 if given sufficient time to train.
 
-If running tensorboard, you'll notice a separate group of scalars named `offpolicy` with losses, approximate frame rate
-and other tracked values in addition to the standard `train` used for on-policy training.
+If running tensorboard, you'll notice a separate group of scalars named `train-offpolicy-losses` and
+`train-offpolicy-misc` with losses, approximate "experiences per second" (i.e. the number of off-policy experiences/steps
+being used to update the model per second), and other tracked values in addition to the standard `train-onpolicy-*`
+used for on-policy training. In the `train-metrics` and `train-misc` sections you'll find the metrics
+quantifying the performance of the agent throughout training and some other plots showing training details.
+*Note that the x-axis for these plots is different than for the `train-offpolicy-*` sections*. This
+is because these plots use the number of rollout steps as the x-axis (i.e. steps that the trained agent
+takes interactively) while the `train-offpolicy-*` plots uses the number of offpolicy "experiences" that have
+been shown to the agent.
+
 
-A view of the training progress about 5 minutes after starting on a CUDA-capable GPU should look similar to
+A view of the training progress about 5 hours after starting on a CUDA-capable GPU should look similar to the below
```

Comment:
5 hours ? :O

Reply:
Haha, I let it train for longer (as a sanity check), in our old tutorial we didn't let it train to completion.